postgresql/Readme.md

```diff
@@ -0,0 +1,30 @@
+# Postgres
+## Psql
```

Comment:
Split this into `## Build` and `## Run`.

Reply:
Resolved with 4ad64acfac7212ef449f971b687716b36e647de6